crates/router/src/connector/aci/transformers.rs

```diff
@@ -203,6 +311,27 @@ pub struct ErrorParameters {
     pub(super) message: String,
 }
 
+fn method_data(redirection_data: AciRedirectionData) -> services::Method {
+    let mut three_ds: bool = false;
```

Comment:
My thought process:   
If there exist redirection link in the PaymentsResponse, I call the `services::RedirectForm`. Since this requires a `method` to be passed and the availability of this `method` is a bit different on 3DS flow and bank redirects. Shown below:
```json
"_comment": "Bank Redirects response",
"redirect":{
    "url":"redirect url comes here",
    "method":"POST",
    "parameters":[
    ]
  },
```
```json
"_comment": "3DS Intermediate response",
{
    "redirect": {
        "url": "https://eu-test.oppwa.com/v1/threeDSecure/execute",
        "parameters": [{
            "name": "name",
            "value": "value"
        }],
        "preconditions": [{
            "origin": "iframe#hidden",
            "waitUntil": "iframe#onload",
            "description": "Hidden iframe post for 3D Secure 2.0",
            "method": "POST",
            "url": "methodURL",
            "parameters": [{
                "name": "threeDSMethodData",
                "value": "methodData"
            }]
        }]
    },
}
```
The code is modified and that `bool` is removed.
```rust
fn method_data(redirection_data: AciRedirectionData) -> services::Method {
    // Check if method exists in 3DS
    if let Some(method_param) = redirection_data
        .parameters
        .iter()
        .find(|param| param.name == *"method")
    {
        // Parse the parameter value as Method enum
        if let Ok(method) = &method_param.value.parse::<services::Method>() {
            return *method;
        }
    }
    redirection_data.method.unwrap_or(services::Method::Post)
}
```

If `redirection_data` is found in 3DS, I return the method given by 3DS, else, I'll be returning `Post` method as default for bank redirects flow which they've explicitly mentioned [here](https://docs.aciworldwide.com/tutorials/server-to-server).